install/roles/well-known-services/templates/nginx.conf

```diff
@@ -50,7 +50,7 @@ server {
 {% if website.install %}
     # Default web site: redirect to www.{{ network.domain }}
     location / {
-        rewrite ^ https://www.{{ network.domain }} permanent;
+        rewrite ^ https://www.{{ network.domain }}$request_uri? permanent;
```

Comment:
We should use a 301 in all those cases, like in #354. There are more in other files.